pkg/git/WebhookEventParser.go

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2020 Devtron Labs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package git
+
+import (
+	"github.com/devtron-labs/git-sensor/internal/sql"
+	"github.com/tidwall/gjson"
+	"go.uber.org/zap"
+	"time"
+)
+
+type WebhookEventParser interface {
+	ParseEvent(selectors []*sql.GitHostWebhookEventSelectors, requestPayloadJson string) (*sql.WebhookEventParsedData, map[string]string, error)
+}
+
+type WebhookEventParserImpl struct {
+	logger *zap.SugaredLogger
+}
+
+func NewWebhookEventParserImpl(logger *zap.SugaredLogger) *WebhookEventParserImpl {
+	return &WebhookEventParserImpl{
+		logger: logger,
+	}
+}
+
+
+const (
+	WEBHOOK_SELECTOR_UNIQUE_ID_NAME string = "unique id"
```

Comment:
generally we try to avoid whitespaces in string constant unless enforced by external system.

Reply:
It would be shown to user when config would be exposed. So it should be user friendly name